docusaurus/docs/iOS/02-tutorials/03-livestream.mdx

```diff
@@ -5,328 +5,325 @@ description: How to build a livestream experience using Stream's video SDKs
 
 ### Livestream Quickstart
 
-In this quickstart we'll build a livestreaming experience that's similar to Twitch.
+In this tutorial we'll quickly build a low-latency in-app livestreaming experience. The livestream is broadcasted using Stream's edge network of servers around the world. We'll cover the following topics:
 
-- The livestream will run on Stream's Edge network of servers around the world
-- You can use ultra low latency (WebRTC) based livestreaming or HLS (slight delay, but better buffering)
-- We'll show how to publish to the livestream from your phone
-- Stream enables you to scale to millions of viewers
-- You can have multiple active participants in the livestream
+- Ultra low latency streaming
+- Multiple streams & co-hosts
+- RTMP in and WebRTC input
+- Exporting to HLS
+- Reactions, custom events and chat
+- Recording & Transcriptions
 
-Let's get started. If you want to see a fully fledged example, check out this [sample app](https://github.com/GetStream/stream-video-ios-examples/tree/main/LivestreamingSample).
+Let's get started, if you have any questions or feedback be sure to let us know via the feedback button.
 
-### Step 1 - Prep
+### Step 1 - Create a new project in Xcode
 
-Before going through this tutorial, please make sure that you have the required setup:
-- `StreamVideo` SDK added to your project
-- Auth setup and user login
-- Required permissions for camera and microphone
+Note that this tutorial was written using **XCode 14**. Setup steps can vary slightly across Xcode versions.
+We recommend using Xcode 14 or newer.
 
-### Step 2 - Publishing your livestream
+1. Create a new project
+2. Select iOS -> App in the new project template
+3. Select SwiftUI as the interface for your app
+4. Name your project **LivestreamSampleApp**.
 
-In a livestream experience, there are two types of users: the ones that stream their content, and the ones that join in and watch the content.
+### Step 2 - Install the SDK & Setup the client
```

Comment:
Shall we copy the structure we have on the other tutorials (check step 2 on https://getstream.io/video/docs/ios/tutorials/audio-room/)

Reply:
done